profile/src/main/java/com/exalt/profile/viewobjects/UserState.kt

```diff
@@ -0,0 +1,7 @@
+package com.exalt.profile.viewobjects
+
+data class UserState(
+    var isLoading: Boolean = false,
+    val data: User? = null,
+    val message: String = ""
+)
```

Comment:
Je pense que le mapping avec la sealed class Loading, Success Error pourrait être une meilleure idée puisque ces states sont séparés comme ici : https://developer.android.com/kotlin/flow/stateflow-and-sharedflow